lib/runnable.js

```diff
@@ -450,3 +452,20 @@ Runnable.prototype._timeoutError = function(ms) {
   }
   return new Error(msg);
 };
+
+/**
+ * Generate an undefined error if `err` is not defined.
+ *
```

Comment:
```js
 * @static
```

Reply:
it's implied; `@static` is necessary if it's not obviously static to jsdoc (which it is)